src/types.h

```diff
@@ -24,9 +24,10 @@ SPDX-License-Identifier: LGPL-2.1-or-later
  * Not all info is added at the same time.
  */
 typedef struct {
+    char outname[33];		  // Up to 10^32 - 1
```

Comment:
make it 16, if it's an array.
I'd make it a `char *` and assign the result of a `strdup`, but a `char []` is still fine.